releases/v20.1.0-alpha.20191118.md

```diff
@@ -0,0 +1,179 @@
+---
+title: What&#39;s New in v20.1.0-alpha.20191118
+toc: true
+summary: Additions and changes in CockroachDB version v20.1.0-alpha.20191118 since version v19.2.0
+---
+
+## November 18, 2019
+
+Get future release notes emailed to you:
+
+<div class="hubspot-install-form install-form-1 clearfix">
+    <script>
+        hbspt.forms.create({
+            css: '',
+            cssClass: 'install-form',
+            portalId: '1753393',
+            formId: '39686297-81d2-45e7-a73f-55a596a8d5ff',
+            formInstanceId: 1,
+            target: '.install-form-1'
+        });
+    </script>
+</div>
+
+### Downloads
+
+<div id="os-tabs" class="clearfix">
+    <a href="https://binaries.cockroachdb.com/cockroach-v20.1.0-alpha.20191118.darwin-10.9-amd64.tgz"><button id="mac" data-eventcategory="mac-binary-release-notes">Mac</button></a>
+    <a href="https://binaries.cockroachdb.com/cockroach-v20.1.0-alpha.20191118.linux-amd64.tgz"><button id="linux" data-eventcategory="linux-binary-release-notes">Linux</button></a>
+    <a href="https://binaries.cockroachdb.com/cockroach-v20.1.0-alpha.20191118.src.tgz"><button id="source" data-eventcategory="source-release-notes">Source</button></a>
+</div>
+
+### Docker image
+
+{% include copy-clipboard.html %}
+~~~shell
+$ docker pull cockroachdb/cockroach-unstable:v20.1.0-alpha.20191118
+~~~
+
+### General changes
+
+- Guidance on [how to contribute to CockroachDB](https://cockroachlabs.atlassian.net/wiki/x/QQFdB) have been moved to the public wiki at wiki.crdb.io. [#41542][#41542]
+- Removed the `kv.bulk_io_write.addsstable_max_rate` [cluster setting](../20.1/cluster-settings.html). [#41745][#41745] {% comment %}doc{% endcomment %}
+- Improved the consistency checker's log output. [#41893][#41893]
+- When the replicas within a range are found to been corrupted, the outliers will be terminated. Previously, the leaseholder replica would terminate, regardless of which replicas disagreed with each other. This is expected to curb the spreading of corrupted data better than the previous approach. [#41902][#41902]
```

Comment:
"are found to have been corrupted" or maybe "are found to be corrupted"?
"curb the spread of corrupted data"

Reply:
Done.